cfme/control/explorer/policies.py

```diff
@@ -354,7 +349,7 @@ def assign_events(self, *events, **kwargs):
             events += self.assigned_events
         view = navigate_to(self, "Details")
         view.configuration.item_select("Edit this Policy's Event assignments")
-        view = self.create_view(EditPolicyEventAssignments)
+        view = self.create_view(EditPolicyEventAssignments, wait="5s")
         assert view.is_displayed
```

Comment:
I suspect you don't need this assert since you wait for view above

Reply:
good point, I will remove that